trestle/core/commands/author/profile.py

```diff
@@ -157,19 +188,31 @@ def _init_arguments(self) -> None:
         output_help_str = 'Name of the output generated json Profile (ok to overwrite original)'
         self.add_argument('-o', '--output', help=output_help_str, required=True, type=str)
         self.add_argument('-sp', '--set-parameters', action='store_true', help=const.HELP_SET_PARAMS, required=False)
+        self.add_argument('-r', '--regenerate', action='store_true', help=const.HELP_REGENERATE)
+        self.add_argument('-vn', '--version', help=const.HELP_VERSION, required=False, type=str)
+        self.add_argument('-rs', '--required-sections', help=const.HELP_REQUIRED_SECTIONS, required=False, type=str)
 
     def _run(self, args: argparse.Namespace) -> int:
         try:
             log.set_log_level_from_args(args)
             trestle_root = pathlib.Path(args.trestle_root)
             # the original profile model name defaults to being the same as the new one
             prof_name = args.output if not args.name else args.name
+            version: Optional[str] = None
+            if 'version' in args and args.version:
```

Comment:
Same as above the first check is not needed

Reply:
issue #1031